game/startup.js

```diff
@@ -0,0 +1,21 @@
+const events = require('./events.js');
+const { Client } = require('discord.js');
+
+module.exports = { StartUp };
+
+async function StartUp(client) {
+    try {
+        const query = await client.sequelize.models.Channel.findAll();
+        query.forEach(async record => {
+            const channel = await client.channels.cache.get(record.dataValues.id);
+            if(channel) {
+                events.OnStart(channel, client);
+            } else {
+                console.log("Channel isn't available (probably because the channel was deleted)."); // TODO: Better managing of this, and probably deleting it from the database.
```

Comment:
Maybe log the channel id as well. Removing it from the database is a good idea, also don't forget to remove all the words that are linked to this channel